postgresql.conf.jinja

```diff
@@ -316,6 +316,33 @@ log_min_duration_statement = '1500ms'
 log_line_prefix = '%m [%p] %q%u@%d '
 log_timezone = 'Europe/Berlin'
 
+# Controls which SQL statements are logged. Valid values are none (off), ddl, mod, and all (all statements).
+# ddl logs all data definition statements, such as CREATE, ALTER, and DROP statements. mod logs all ddl
+# statements, plus data-modifying statements such as INSERT, UPDATE, DELETE, TRUNCATE, and COPY FROM. PREPARE, 
+# EXECUTE, and EXPLAIN ANALYZE statements are also logged if their contained command is of an appropriate type. 
+{% if log_statement is defined %}
+log_statement = '{{ log_statement }}'
+{% else %}
+log_statement = 'none'
+{% endif %}
+
+# This parameter enables the logging collector, which is a background process that captures log
+# messages sent to stderr and redirects them into log files.
+{% if logging_collector %}
+logging_collector = on
```

Comment:
Good idea to nest the `log_*` settings that are depending on `logging_collector` being enabled!

Reply:
I think it might be wise to add the [two notes](https://www.postgresql.org/docs/11/runtime-config-logging.html) in the documentation to the readme, as `logging_collector` might have some serious implicatons.